### DISCUSSION TITLE
Error: Your `getStaticProps` function did not return an object. Did you forget to add a `return`?

### DISCUSSION THREAD

**Discussion by MohdAdilMohammedAkbar** [piyushgarg-dev/piyushgargdev-nextjs#1316] [General]
What is This Error ?
Can anyone Help me ,I am not able to run it locally

**Comment by prathamesh-pichkate** [comment]
@piyushgarg-dev I am also getting the same error and  not able to fix it .

Error:
Server Error
Error: Your `getStaticProps` function did not return an object. Did you forget to add a `return`?

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Call Stack
renderToHTML
file:///home/prathamesh/Desktop/open-sorce/piyushgargdev-nextjs/node_modules/next/dist/server/render.js (463:19)
process.processTicksAndRejections
node:internal/process/task_queues (95:5)
async PagesRouteModule.render
node_modules/next/dist/server/future/route-modules/pages/module.js (33:0)
async doRender
file:///home/prathamesh/Desktop/open-sorce/piyushgargdev-nextjs/node_modules/next/dist/server/base-server.js (1051:26)
async cacheEntry.responseCache.get.incrementalCache.incrementalCache
file:///home/prathamesh/Desktop/open-sorce/piyushgargdev-nextjs/node_modules/next/dist/server/base-server.js (1183:28)
async
file:///home/prathamesh/Desktop/open-sorce/piyushgargdev-nextjs/node_modules/next/dist/server/response-cache/index.js (99:36)
